<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureFilesystem</class>
 <widget class="QWidget" name="ConfigureFilesystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Filesystem</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Storage Directories</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>NAND</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="nand_directory_button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="nand_directory_edit"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="sdmc_directory_edit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>SD Card</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="sdmc_directory_button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Gamecard</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="gamecard_path_edit"/>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="gamecard_inserted">
          <property name="text">
           <string>Inserted</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="gamecard_current_game">
          <property name="text">
           <string>Current Game</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="gamecard_path_button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Patch Manager</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QLineEdit" name="load_path_edit"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="dump_path_edit"/>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="dump_path_button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="load_path_button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="dump_nso">
            <property name="text">
             <string>Dump Decompressed NSOs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dump_exefs">
            <property name="text">
             <string>Dump ExeFS</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Mod Load Root</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Dump Root</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Caching</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="cache_game_list">
            <property name="text">
             <string>Cache Game List Metadata</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reset_game_list_cache">
            <property name="text">
             <string>Reset Metadata Cache</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
